nav2_bringup/launch/bringup_launch.py

```diff
@@ -58,16 +57,6 @@ def generate_launch_description():
     #              https://github.com/ros2/launch_ros/issues/56
     remappings = [('/tf', 'tf'), ('/tf_static', 'tf_static')]
 
-    # Only it applys when `use_namespace` is True.
-    # '<robot_namespace>' keyword shall be replaced by 'namespace' launch argument
-    # in config file 'nav2_multirobot_params.yaml' as a default & example.
-    # User defined config file should contain '<robot_namespace>' keyword for the replacements.
-    params_file = ReplaceString(
-        source_file=params_file,
-        replacements={'<robot_namespace>': ('/', namespace)},
-        condition=IfCondition(use_namespace),
-    )
-
     configured_params = ParameterFile(
```

Comment:
I can't quite tell if this is still necessary or not now that the above was removed

Reply:
The `ParameterFile` bit? Yes, that is necessary